users/index.html

```diff
@@ -12,14 +12,41 @@
     <script src="/users/utils.js" defer></script>
     <script type="module" src="/users/script.js" defer></script>
   </head>
+
   <body>
     <header class="header">
       <h1>User Management</h1>
     </header>
     <section id="main-section">
-      <div id="search">
-        <input id="user-search" type="text" placeholder="Search Users" />
+      <div class="search-filter">
+        <div id="search">
+          <input id="user-search" type="text" placeholder="Search Users" />
+        </div>
+        <div class="filter-button-container">
+          <button id="filter-button" class="filter-button">Filters</button>
+        </div>
```

Comment:
why the button is wrapped around a div, what's the use of `filter-button-container` div

Reply:
for set the position of button to right btw i removed the `filter-button-container` and set position property to button